### PR TITLE
📖 Commit signing off update.

### PR DESCRIPTION
**Pull request by adilGhaffarDev**
**What this PR does / why we need it**:
Updating contributing guide regarding signing off of commits

### CODE REVIEW

**Comment by adilGhaffarDev** [comment]
@tuminoid @kashifest

**Comment by adilGhaffarDev** [comment]
cc @tuminoid @kashifest

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,24 @@ By contributing to this project you agree to the Developer Certificate of Origin
 statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
+### Git commit Sign-off
+
+This line should be included at the bottom of git commit message.
+
```
Comment (smoshiur1237) [inline CONTRIBUTING.md:41]: Can you please rewrite the line? You can have something like this or just rephrase it: “Commit message should contain signed off section with full name and email. For example:”

Comment (smoshiur1237) [reply CONTRIBUTING.md:41]: Otherwise it sounds like John Doe should be the fixed name

Comment (adilGhaffarDev) [reply CONTRIBUTING.md:41]: I will change it.

**Review by lentzi90 — COMMENTED** [review]
/approve

**Pull request merged**